pages/400.js

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link'
+
+export default function Error400() {
+  return (
+    <div>
+      <h1>400 Error - Bad request error occurred</h1>
+      <Link href="/dashboard">
+        <a className="text-sm hover:text-bright-blue-solid visited:text-purple-solid underline">
+          Return to dashboard main page
```

Comment:
I think it's better to move it to locales/en.js file

Reply:
done in b18b6c06469fc4da56ab4cde1926e2e1275d6e73